pkgs/top-level/aliases.nix

```diff
@@ -597,6 +597,7 @@ mapAliases ({
   nix_2_4 = nixVersions.nix_2_4;
   nix_2_5 = nixVersions.nix_2_5;
   nix_2_6 = nixVersions.nix_2_6;
+  nixops = throw "'nixops' has been removed. Please use 'nixops_unstable' for the time being."; # Added 2023-10-26
```

Comment:
```suggestion
  nixops = throw "'nixops' has been removed. Please use 'nixops_unstable' for the time being."; # Added 2023-10-26
  nixops-dns = throw "...";
```

Reply:
Actually, maybe we can keep nixops-dns? I'm not sure if the state file format has changed.

Reply:
At a glance, the state format has not changed, so it's OK to keep. I've moved it to `pkgs/by-name` though, because now it definitely has no business being in package-management.